client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -58,29 +119,43 @@ function EmailPlanMailboxesList( { emails, isLoadingEmails } ) {
 		);
 	}
 
-	const emailsItems = emails.map( ( email ) => {
+	const mailboxItems = mailboxes.map( ( mailbox ) => {
+		const { action, warning } = getActionsForMailbox( mailbox, translate, dispatch );
+
 		return (
-			<MailboxListItem key={ email.mailbox }>
-				<MaterialIcon icon="email" />
-				<span>
-					{ email.mailbox }@{ email.domain }
-				</span>
-				{ isEmailUserAdmin( email ) && (
+			<MailboxListItem key={ mailbox.mailbox } isError={ !! warning }>
+				<div className="email-plan-mailboxes-list__mailbox-list-item-main">
+					<div>
+						<MaterialIcon icon="email" />
+						<span>
+							{ mailbox.mailbox }@{ mailbox.domain }
+						</span>
+					</div>
+					{ isEmailForward( mailbox ) && (
```

Comment:
It might be worth it to unpack this into a function so that this is more readable. It'll soon get really cluttered in here as we add more service specific actions, and I know this is only specific to forwards but it will make things really simple to glean.

Reply:
Done!